src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -3583,4 +3594,12 @@ public void addBatch(String sql) throws SQLServerException {
         Object[] msgArgs = {"addBatch()"};
         throw new SQLServerException(this, form.format(msgArgs), null, 0, false);
     }
+
+    private void clearPrepStmtHandle(){
+        prepStmtHandle = 0;
+        cachedPreparedStatementHandle = null;
+        if(getStatementLogger().isLoggable(Level.FINER)){
```

Comment:
Minor formatting issue. Should have a space preceeding the initila if statement brace and after the closing brace

Reply:
```suggestion
        if (getStatementLogger().isLoggable(Level.FINER)){
```

Reply:
pls run formatter on all files